be/src/olap/rowset/beta_rowset.cpp

```diff
@@ -124,6 +125,21 @@ Status BetaRowset::get_inverted_index_size(size_t* index_size) {
     return Status::OK();
 }
 
+void BetaRowset::clear_inverted_index_cache() {
```

Comment:
Is there any difference with BetaRowset::remove()?

Reply:
remove() delets files too. When a tablet is moved to trash, the files are not deleted immediately. I let remove call clear_cache to reduce duplicated code.